### ISSUE TITLE
TimeGate is not reporting the HTTP status correctly

### ISSUE THREAD

**Issue by anjackson** [ukwa/ukwa-pywb#58]
This is related to #44, #45 but comments there indicate that the TimeGate used to report the status correctly. This seems not to be the case now:

```
# curl -I https://beta.webarchive.org.uk/wayback/archive/http://www.sahahdsf
HTTP/1.1 200 OK
Server: nginx/1.16.1
Date: Mon, 02 Mar 2020 16:10:51 GMT
Content-Type: text/html
Content-Length: 3065
Connection: keep-alive
Link: <http://www.sahahdsf>; rel="original", <https://beta.webarchive.org.uk/wayback/archive/http://www.sahahdsf>; rel="timegate", <https://beta.webarchive.org.uk/wayback/archive/timemap/link/http://www.sahahdsf>; rel="timemap"; type="application/link-format"
Vary: accept-datetime
Accept-Ranges: bytes
```

This causes problems with the Memento Aggregator. I think even if we skip it everywhere else, the TimeGate URI should do a lookup and return the right status code.

**Comment by anjackson** [comment]
The problem with using the `mp_/` modifier is that is represents a _breaking change of our TimeGate API for all clients_ (a basic assumption of our work moving from OWB was that switching to pywb should be API compatible).  We use the API in various places, as well as it being used by third-parties, and this breaking change has caused a range of problems with both internal and external systems.

This is why I would prefer it if the TimeGate URL `https://www.webarchive.org.uk/wayback/archive/<QUERY_URL>` behaved in the same way as the `mp_/` version.

**Comment by ikreymer** [comment]
The expected timegate behavior should now work w/o the `mp_` modifier. This is also part of the just released pywb 2.4.0
